docs/src/utils/openInStackblitz.ts

```diff
@@ -0,0 +1,53 @@
+import stackBlitz from '@stackblitz/sdk';
+
+interface StackBlitzProps {
+  title?: string;
+  description?: string;
+  name?: string;
+  text?: string;
+}
+
+export const openReactStackBlitz = ({
+  title = 'Amplify UI',
+  description = 'Amplify UI example',
+  name,
+  text,
+}: StackBlitzProps) => {
+  const fileName = `src/${name}.js`;
+
+  stackBlitz.openProject(
+    {
+      files: {
+        [fileName]: `import * as React from 'react';\n${text}`,
```

Comment:
Adding `import * as React from 'react';` to the file contents, should we just be adding this to our example code instead?